k8s/upgrades/README.md

```diff
@@ -7,6 +7,9 @@ This document describes the steps for the following OpenEBS Upgrade paths:
 - Upgrade from 1.0.0 to 1.1.0
```

Comment:
This section can be updated/simplified as follows:

- Upgrade from 1.0.0 to any of 1.1.0, 1.2.0, 1.3.0
- Upgrade from 1.1.0 to any of 1.2.0, 1.3.0
- Upgrade from 1.2.0 to 1.3.0

Reply:
Done